### PR TITLE
switch cloudstorage out for s3filestore

### PR DESCRIPTION
**Pull request by chris48s**
This PR switches us from using okfn/ckanext-cloudstorage to keitaroinc/ckanext-s3filestore. Fairly minimal changes going on here but I've also taken the opportunity to get rid of a pylons controller and replace it with a flask blueprint

Refs https://github.com/okfn/docker-ckan-unhcr-aws/pull/29
Refs https://github.com/okfn/ckanext-unhcr/issues/399#issuecomment-797575581

### CODE REVIEW

File: ckanext/unhcr/plugin.py [hunk 1]
```diff
@@ -223,11 +223,7 @@ def before_map(self, _map):
             '/deposited-dataset/{id}/resource/{resource_id}/download',
             '/deposited-dataset/{id}/resource/{resource_id}/download/{filename}',
         ]
-        if not plugins.plugin_loaded('cloudstorage'):
-            for route in download_routes:
-                _map.connect(route, controller=controller, action='resource_download')
-        else:
-            controller='ckanext.unhcr.controllers.extended_storage:ExtendedStorageController'
+        if not plugins.plugin_loaded('s3filestore'):
```
Comment (amercader) [inline ckanext/unhcr/plugin.py:226]: We don't need to add these Pylons routes any more now that we have a blueprint right?

**Review by amercader — APPROVED** [review]
@chris48s just one minor comment about routes, the rest looks great

File: ckanext/unhcr/plugin.py (hunk 1 above)
Comment (chris48s) [reply ckanext/unhcr/plugin.py:226]: This block here

```py
if not plugins.plugin_loaded('s3filestore'):
    for route in download_routes:
        _map.connect(route, controller=controller, action='resource_download')
```

is adding the fallback routes for local storage. It is just there so we can develop using the local storage backend - you don't have to have an S3 bucket for your dev copy (although personally I do use a S3 bucket for my dev copy). This is still a pylons controller.


Further down in `get_blueprint()` where I am saying

```py
if plugins.plugin_loaded('s3filestore'):
    bp.append(blueprints.unhcr_s3_resource_blueprint)
```

that is adding the new blueprint for when we are using the S3 backend.

Comment (amercader) [reply ckanext/unhcr/plugin.py:226]: Got you

**Pull request merged**